package.json

```diff
@@ -64,12 +64,16 @@
     "eslint-plugin-react": "^7.14.3",
     "husky": "^3.0.4",
     "jest": "^24.9.0",
+    "lint-staged": "^10.0.0-1",
     "prettier": "^1.18.2",
     "pretty-quick": "^1.11.1"
   },
   "husky": {
     "hooks": {
-      "pre-commit": "yarn format-staged && yarn lint"
+      "pre-commit": "yarn format-staged && lint-staged"
     }
+  },
+  "lint-staged": {
+    "*.{js,jsx}": "eslint"
```

Comment:
Please notice `eslint` is being run on staged files only at pre-commit hook! CI will continue to eslint everything under src/ + pages/

Reply:
👍